src/main/webapp/app/modules/auth/email-login/auth-email-login.tsx

```diff
@@ -67,7 +86,7 @@ export class AuthEmailLogin extends React.Component<ILoginProps> {
                 <Link to="/auth/email/register" className="text-decoration-none">
                   No account? Create now
                 </Link>
-                <Button type="submit" className="w-100 mt-4">
+                <Button type="submit" className="w-100 mt-4" disabled={!this.state.isSubmitBtnEnabled}>
```

Comment:
fixed in 04a531dab3b6ea081f90024da890374868a04d40

I have changed the naming to `isSubmitting`
the reason why I did not use `isDisabled` is that I avoid naming my variable in a 'negative' manner.
Just my personal preference.

Reply:
@kachun333 agree with your concern and preference 👍 
isSubmitting is nice 💯 
suggest to change from enabled because I miss out the not in the beginning of review 😃

Reply:
![goodjob](https://user-images.githubusercontent.com/35193728/110246068-01b47d00-7fa1-11eb-8c41-f4584bf29989.jpg)